### DISCUSSION TITLE
Format Ant Design Time Picker Response Directly When Selecting a Time

### DISCUSSION THREAD

**Discussion by opuopu** [ant-design/ant-design#48772] [General]
Hello good people. when we select a time from ant design it gives the result by this format: 
![image](https://github.com/ant-design/ant-design/assets/43672649/ad535d30-b95a-4b2e-997d-d99c73cd1017)
but i need it directly HH:mm format. i know we can format it manually when submitting a form. but i need something directly when i select time. is that any way to handle this?

**Comment by crazyair** [comment]
https://ant.design/docs/blog/form-names

**Reply by opuopu** [reply]
i dont understand

**Reply by crazyair** [reply]
https://stackblitz.com/edit/react-jwdpiv?file=demo.tsx
